lib/routes/main_page.dart

```diff
@@ -319,6 +319,8 @@ class _MainPageState extends State<MainPage> with TickerProviderStateMixin {
     }
   }
   Future<void> sync() async {
```

Comment:
Also you can remove Future<void> and use directly void

Reply:
Just tried removing Future but it gives an error since the onRefresh: property only accepts Future<void> functions

Reply:
Right, then leave as is